src/test/pybind/test_rbd.py

```diff
@@ -1854,11 +1854,14 @@ def test_break_lock(self):
             try:
                 rados2.conf_set('rbd_blacklist_on_break_lock', 'true')
                 with Image(ioctx2, image_name) as image, \
-                     Image(blacklist_ioctx, image_name) as blacklist_image:
+                    Image(blacklist_ioctx, image_name) as blacklist_image:
```

Comment:
nit: I am not sure I like you changed the indentation here. Now it looks like this line is a part of the block below, while it is a continuation of the previous line.

Reply:
oh ,I'll restore it